test/tmemleak

```diff
@@ -0,0 +1,8 @@
+#!/bin/sh
+
+../util/shlib_wrap.sh ./memleaktest freeit || exit 1
```

Comment:
I haven't gotten to dig into OpenSSL::test yet, so take this with a grain of salt, but I don't see anything that actually calls tmemleak --  test/recipes/90-test_memleak.t looks to be calling the memleak executable directly (and not in an interesting way).